### PR TITLE
[ADD] l10n_ar_base: Add new Argentinian Localization Base module

### PR DESCRIPTION
**Pull request by zaoral**
This PR adds the l10n_ar_base module that holds all the basic information for Argentinian Localization (Do not include account)

### CODE REVIEW

**Comment by zaoral** [comment]
@ren-odoo @jco-odoo @jjscarafia

Hello Rubi and Josse. Here is the PR of l10n_ar_base module with the changes we agreed ready for review. This PR supersed https://github.com/odoo/odoo/pull/32963 (we were not able to change the old PR head repository)

NOTE: About the topic "The VAT field is visible and we will retrieve the information of the identification number and vice versa." We implemented this vise versa functionality and it works. You can check the code in this [commit](https://github.com/odoo-dev/odoo/pull/129/commits/82c3ec39ff3a6a4e8eb11cc5bf163affa8b87750), but after we analyzed the results we have second thoughts about this functionality and we decided to remove  the integration between the  vat and id_number/id_category fields. 

These are the reasons:

* adds unnecessary complexity to the code
* add unnecessary complexity to the user,  if the user modify one of the field the other one will be updated in some cases and others not and viceversa.
* using Odoo's vat field  number do not lets us to use a different cuit number for a commercial partner that relationship parent/child (the case we discuss about the main company and the shops).
* we are aware that maybe this one will not be needed thanks to the improves about hide/shown fields taking into account the localization.
* also we are aware that Odoo is considering to define a better way to manage multiple identification numbers.

In Conclusion: 
This integration it doesn't add any value functional or technical 

Please let us know your thoughts about this

Best Regards

**Comment by jco-odoo** [comment]
@zaoral Ok, it is more important to continue on the many other things.  We have already had some discussion about this too (general invisible field / calculated field for the identification number based on VAT or identification field /...) ,  but no single solution yet.  The parent/child with different vats is however something we would like to avoid.

**Comment by zaoral** [comment]
@jco-odoo 

We are agreed there are many more important things: only trying to communicate when something in our side change.

Are you agreed with this PR as it is in order to merge? or would you like to add again the update-inverse functionality from [commit](https://github.com/odoo-dev/odoo/pull/129/commits/82c3ec39ff3a6a4e8eb11cc5bf163affa8b87750)
